docs/docs/configuration/hardware_acceleration.md

```diff
@@ -26,24 +28,50 @@ ffmpeg:
 
 **NOTICE**: With some of the processors, like the J4125, the default driver `iHD` doesn't seem to work correctly for hardware acceleration. You may need to change the driver to `i965` by adding the following environment variable `LIBVA_DRIVER_NAME=i965` to your docker-compose file or [in the frigate.yml for HA OS users](advanced.md#environment_vars).
 
-### Intel-based CPUs (>=10th Generation) via Quicksync
+#### Intel-based CPUs (>=10th Generation) via Quicksync
 
 QSV must be set specifically based on the video encoding of the stream.
 
-#### H.264 streams
+##### H.264 streams
 
 ```yaml
 ffmpeg:
   hwaccel_args: preset-intel-qsv-h264
 ```
 
-#### H.265 streams
+##### H.265 streams
 
 ```yaml
 ffmpeg:
   hwaccel_args: preset-intel-qsv-h265
 ```
 
+#### Docker Configuration
+
+Additional configuration is needed for the Docker container to be able to access the `intel_gpu_top` command for GPU stats. Two possible changes need to be made: 1) Adding the `CAP_PERFMON` capability and 2) Setting the `perf_event_paranoid` low enough to allow access to the perfomance event system.
+
+##### CAP_PERFMON
+
+Only recent versions of Docker support the `CAP_PERFMON` capability. You can test to see if yours supports it by running: `docker run --cap-add=CAP_PERFMON hello-world`
+
+Pass `cap-add=CAP_PERFMON` to the container. With docker compose, that looks like this:
+
+```yaml
+services:
+  frigate:
+    ...
+      cap_add:
+        - CAP_PERFMON
+```
```

Comment:
I would refactor this to show specific examples of both similar to how the nvidia docs show docker compose and docker cli examples